js/digifinex.js

```diff
@@ -330,52 +323,13 @@ module.exports = class digifinex extends Exchange {
             const deposit = depositStatus > 0;
             const withdraw = withdrawStatus > 0;
             const active = deposit && withdraw;
-            const fee = this.safeNumber (currency, 'min_withdraw_fee'); // withdraw_fee_rate was zero for all currencies, so this was the worst case scenario
```

Comment:
@Dan-krm are these `fetchCurrencies` edits meant to be here?

Reply:
No I don't think so